### PR TITLE
Fix: High Security fix

### PR DESCRIPTION
**Pull request by agg3fe**
## Description
Fixing high security findings

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -68,6 +68,7 @@
       <spring.boot.version>3.2.4</spring.boot.version>
       <spring-framework.version>6.1.6</spring-framework.version>
       <lombok.version>1.18.24</lombok.version>
+      <tomcat.embed.version>10.1.25</tomcat.embed.version>
```
[the remark this answers is not in the document]
Comment (agg3fe) [reply pom.xml:71]: I tried to update spring boot itself, but that did not change the tomcat version. so I had to update tomcat separately.

[the remark this answers is not in the document]
Comment (agg3fe) [reply pom.xml:71]: I think I found something. let me update it.

**Review by tunacicek — APPROVED** [review]
LGTM

**Review by tunacicek — APPROVED** [review]
LGTM

**Pull request merged**